src/desktop/apps/artwork/components/auction/templates/buy_now.jade

```diff
@@ -6,6 +6,6 @@ if artwork.is_buy_nowable
     .artwork-auction__buy-now__amount
       = artwork.sale_message
 
-    form.artwork-auction__buy-now__form
+    form.artwork-auction__buy-now__form.js-artwork-auction-buy-now-form
```

Comment:
Just added to target the form from the auction flow... I think this might not be necessary because I don't think this form actually has the edition set info anywhere on it.